lesson_03/quiz/src/quizzes/lj_mcwilliams_quiz.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  AnswerChoice,
+  MultipleChoiceQuizQuestion,
+  QuizQuestion,
+  QuizQuestionProvider,
+} from 'codedifferently-instructional';
+
+export class LJMcWilliamsQuiz implements QuizQuestionProvider {
```

Comment:
Would be better to name this class `LjMcwilliamsQuiz`. Please change.

Reply:
just updated camel casing